When upgrade aws to 6.18.0 I'm getting plugin did not respond

### Terraform and AWS Provider Version

```console
`
PS D:\Work\GitHub\DevOps\environments\shared-pr1> terraform --version
Terraform v1.13.3
on windows_amd64
+ provider registry.terraform.io/hashicorp/aws v6.18.0
+ provider registry.terraform.io/hashicorp/random v3.7.2
+ provider registry.terraform.io/hashicorp/template v2.2.0

Your version of Terraform is out of date! The latest version
is 1.13.4. You can update by downloading from https://developer.hashicorp.com/terraform/install
`
```

### Affected Resource(s) or Data Source(s)

_No response_

### Expected Behavior

Terraform  to produce correct plan

### Actual Behavior

Terraform fails with "Error: Plugin did not respond" error

### Relevant Error/Panic Output

```console
module.api-gateway.aws_api_gateway_method_response.gapi_reports_options_proxy_response_200: Refreshing state... [id=agmr-ttus0lt429-vk78x6-OPTIONS-200]
module.api-gateway.aws_api_gateway_integration_response.bulk_zoning_data_get_list_response_200: Refreshing state... [id=agir-0xb6udgjki-xeqd3z-GET-200]
module.api-gateway.aws_api_gateway_integration_response.bulk_zoning_data_get_response_200: Refreshing state... [id=agir-0xb6udgjki-s4os35-GET-200]
module.api-gateway.aws_api_gateway_integration_response.gapi_reports_options_proxy_response_200: Refreshing state... [id=agir-ttus0lt429-vk78x6-OPTIONS-200]
module.api-gateway.aws_api_gateway_integration_response.libreoffice_options_proxy_response_200: Refreshing state... [id=agir-c5dbz4mn93-jqp31s-OPTIONS-200]

Planning failed. Terraform encountered an error while generating this plan.

╷
│ Error: Plugin did not respond
│
│   with aws_iam_role.s3_api_gateway_role,
│   on api-gateway.tf line 15, in resource "aws_iam_role" "s3_api_gateway_role":
│   15: resource "aws_iam_role" "s3_api_gateway_role" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ReadResource call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_api_gateway_rest_api.ClientsS3Storage,
│   on api-gateway.tf line 54, in resource "aws_api_gateway_rest_api" "ClientsS3Storage":
│   54: resource "aws_api_gateway_rest_api" "ClientsS3Storage" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with data.aws_secretsmanager_secret_version.current,
│   on batch.tf line 5, in data "aws_secretsmanager_secret_version" "current":
│    5: data "aws_secretsmanager_secret_version" "current" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with data.aws_instance.instance_lookup[3],
│   on cloudwatch.tf line 1, in data "aws_instance" "instance_lookup":
│    1: data "aws_instance" "instance_lookup" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_cloudwatch_log_group.pr1,
│   on cloudwatch.tf line 32, in resource "aws_cloudwatch_log_group" "pr1":
│   32: resource "aws_cloudwatch_log_group" "pr1" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ReadResource call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with data.aws_db_instance.rds_instance_lookup["shared-psql-pr1-tiles1"],
│   on cloudwatch.tf line 250, in data "aws_db_instance" "rds_instance_lookup":
│  250: data "aws_db_instance" "rds_instance_lookup" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ReadDataSource call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_iam_user.corelogic-iam-user,
│   on corelogic.tf line 11, in resource "aws_iam_user" "corelogic-iam-user":
│   11: resource "aws_iam_user" "corelogic-iam-user" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_iam_policy.corelogic-iam-user-policy,
│   on corelogic.tf line 22, in resource "aws_iam_policy" "corelogic-iam-user-policy":
│   22: resource "aws_iam_policy" "corelogic-iam-user-policy" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_iam_role.corelogic-iam-user-role,
│   on corelogic.tf line 73, in resource "aws_iam_role" "corelogic-iam-user-role":
│   73: resource "aws_iam_role" "corelogic-iam-user-role" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with data.aws_iam_policy_document.clients_s3_gateway_storage,
│   on lambda.tf line 30, in data "aws_iam_policy_document" "clients_s3_gateway_storage":
│   30: data "aws_iam_policy_document" "clients_s3_gateway_storage" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with data.aws_iam_policy_document.clients_s3_gateway_storage_policy,
│   on lambda.tf line 55, in data "aws_iam_policy_document" "clients_s3_gateway_storage_policy":
│   55: data "aws_iam_policy_document" "clients_s3_gateway_storage_policy" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with data.aws_iam_policy_document.clients_s3_gateway_storage_s3_access_policy,
│   on lambda.tf line 225, in data "aws_iam_policy_document" "clients_s3_gateway_storage_s3_access_policy":
│  225: data "aws_iam_policy_document" "clients_s3_gateway_storage_s3_access_policy" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_cloudwatch_log_group.clients_s3_gateway_storage,
│   on lambda.tf line 246, in resource "aws_cloudwatch_log_group" "clients_s3_gateway_storage":
│  246: resource "aws_cloudwatch_log_group" "clients_s3_gateway_storage" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with data.aws_ami.base_image,
│   on main.tf line 49, in data "aws_ami" "base_image":
│   49: data "aws_ami" "base_image" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with data.aws_ami.base_image_22,
│   on main.tf line 58, in data "aws_ami" "base_image_22":
│   58: data "aws_ami" "base_image_22" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_ebs_volume.workhouse_import_storage,
│   on main.tf line 129, in resource "aws_ebs_volume" "workhouse_import_storage":
│  129: resource "aws_ebs_volume" "workhouse_import_storage" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_efs_file_system.umt-storage,
│   on main.tf line 225, in resource "aws_efs_file_system" "umt-storage":
│  225: resource "aws_efs_file_system" "umt-storage" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_efs_file_system.apigee-storage,
│   on main.tf line 270, in resource "aws_efs_file_system" "apigee-storage":
│  270: resource "aws_efs_file_system" "apigee-storage" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_efs_file_system.mmt-storage,
│   on main.tf line 317, in resource "aws_efs_file_system" "mmt-storage":
│  317: resource "aws_efs_file_system" "mmt-storage" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ReadResource call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_elasticache_cluster.redis_index1,
│   on main.tf line 481, in resource "aws_elasticache_cluster" "redis_index1":
│  481: resource "aws_elasticache_cluster" "redis_index1" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_elasticache_cluster.redis_gridics,
│   on main.tf line 498, in resource "aws_elasticache_cluster" "redis_gridics":
│  498: resource "aws_elasticache_cluster" "redis_gridics" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_elasticache_cluster.redis_codehub1,
│   on main.tf line 515, in resource "aws_elasticache_cluster" "redis_codehub1":
│  515: resource "aws_elasticache_cluster" "redis_codehub1" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_elasticsearch_domain.pr1-ch1,
│   on main.tf line 532, in resource "aws_elasticsearch_domain" "pr1-ch1":
│  532: resource "aws_elasticsearch_domain" "pr1-ch1" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_s3_bucket.pr1-propzone-storage,
│   on propzone.tf line 118, in resource "aws_s3_bucket" "pr1-propzone-storage":
│  118: resource "aws_s3_bucket" "pr1-propzone-storage" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_cloudfront_origin_access_identity.propzone_origin_access_identity,
│   on propzone.tf line 136, in resource "aws_cloudfront_origin_access_identity" "propzone_origin_access_identity":
│  136: resource "aws_cloudfront_origin_access_identity" "propzone_origin_access_identity" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_efs_file_system.propzone-storage,
│   on propzone.tf line 191, in resource "aws_efs_file_system" "propzone-storage":
│  191: resource "aws_efs_file_system" "propzone-storage" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_s3_bucket.pr1-user-storage,
│   on s3.tf line 1, in resource "aws_s3_bucket" "pr1-user-storage":
│    1: resource "aws_s3_bucket" "pr1-user-storage" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_cloudfront_origin_access_identity.origin_access_identity,
│   on s3.tf line 61, in resource "aws_cloudfront_origin_access_identity" "origin_access_identity":
│   61: resource "aws_cloudfront_origin_access_identity" "origin_access_identity" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_route53_record.s3_userdata,
│   on s3.tf line 257, in resource "aws_route53_record" "s3_userdata":
│  257: resource "aws_route53_record" "s3_userdata" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ReadResource call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_s3_bucket.pr1-zoning-storage,
│   on s3.tf line 265, in resource "aws_s3_bucket" "pr1-zoning-storage":
│  265: resource "aws_s3_bucket" "pr1-zoning-storage" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_cloudfront_origin_access_identity.zoning_origin_access_identity,
│   on s3.tf line 283, in resource "aws_cloudfront_origin_access_identity" "zoning_origin_access_identity":
│  283: resource "aws_cloudfront_origin_access_identity" "zoning_origin_access_identity" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_s3_bucket.gridics-data-delivery,
│   on s3.tf line 364, in resource "aws_s3_bucket" "gridics-data-delivery":
│  364: resource "aws_s3_bucket" "gridics-data-delivery" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with data.aws_iam_policy_document.download-access-gridics-data-delivery,
│   on s3.tf line 406, in data "aws_iam_policy_document" "download-access-gridics-data-delivery":
│  406: data "aws_iam_policy_document" "download-access-gridics-data-delivery" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_iam_policy.s3_policy,
│   on s3.tf line 473, in resource "aws_iam_policy" "s3_policy":
│  473: resource "aws_iam_policy" "s3_policy" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_s3_bucket.clients-s3-storage,
│   on s3.tf line 502, in resource "aws_s3_bucket" "clients-s3-storage":
│  502: resource "aws_s3_bucket" "clients-s3-storage" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_s3_bucket.solr-backups,
│   on solr.tf line 308, in resource "aws_s3_bucket" "solr-backups":
│  308: resource "aws_s3_bucket" "solr-backups" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_alb_target_group.wh-tg,
│   on targets.tf line 19, in resource "aws_alb_target_group" "wh-tg":
│   19: resource "aws_alb_target_group" "wh-tg" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_s3_bucket.pr1-tiles-storage,
│   on tiles1.tf line 1, in resource "aws_s3_bucket" "pr1-tiles-storage":
│    1: resource "aws_s3_bucket" "pr1-tiles-storage" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_cloudfront_origin_access_identity.tiles_origin_access_identity,
│   on tiles1.tf line 31, in resource "aws_cloudfront_origin_access_identity" "tiles_origin_access_identity":
│   31: resource "aws_cloudfront_origin_access_identity" "tiles_origin_access_identity" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with aws_cloudfront_response_headers_policy.VectorTilesCache,
│   on tiles1.tf line 101, in resource "aws_cloudfront_response_headers_policy" "VectorTilesCache":
│  101: resource "aws_cloudfront_response_headers_policy" "VectorTilesCache" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.api.data.aws_ami.api_image,
│   on ..\..\terraform_modules\api\main.tf line 2, in data "aws_ami" "api_image":
│    2: data "aws_ami" "api_image" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.api.data.aws_ami.api_internal_image,
│   on ..\..\terraform_modules\api\main.tf line 10, in data "aws_ami" "api_internal_image":
│   10: data "aws_ami" "api_internal_image" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.api.aws_route53_record.api-internal,
│   on ..\..\terraform_modules\api\main.tf line 22, in resource "aws_route53_record" "api-internal":
│   22: resource "aws_route53_record" "api-internal" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ReadResource call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.api.aws_route53_record.api-main,
│   on ..\..\terraform_modules\api\main.tf line 238, in resource "aws_route53_record" "api-main":
│  238: resource "aws_route53_record" "api-main" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.aws-batch.data.aws_ami.batch_main,
│   on ..\..\terraform_modules\batch\main.tf line 29, in data "aws_ami" "batch_main":
│   29: data "aws_ami" "batch_main" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.aws-batch.aws_launch_template.main,
│   on ..\..\terraform_modules\batch\main.tf line 191, in resource "aws_launch_template" "main":
│  191: resource "aws_launch_template" "main" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.aws-batch.aws_batch_job_definition.job-definition-tiles,
│   on ..\..\terraform_modules\batch\main.tf line 361, in resource "aws_batch_job_definition" "job-definition-tiles":
│  361: resource "aws_batch_job_definition" "job-definition-tiles" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.aws-batch.aws_batch_job_definition.job-definition-user-tiles,
│   on ..\..\terraform_modules\batch\main.tf line 463, in resource "aws_batch_job_definition" "job-definition-user-tiles":
│  463: resource "aws_batch_job_definition" "job-definition-user-tiles" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.cloudfront.aws_cloudfront_origin_access_control.lambda_oac,
│   on ..\..\terraform_modules\cloudfront\main.tf line 1, in resource "aws_cloudfront_origin_access_control" "lambda_oac":
│    1: resource "aws_cloudfront_origin_access_control" "lambda_oac" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.cloudfront.aws_cloudfront_origin_request_policy.dynamic_tiles,
│   on ..\..\terraform_modules\cloudfront\main.tf line 72, in resource "aws_cloudfront_origin_request_policy" "dynamic_tiles":
│   72: resource "aws_cloudfront_origin_request_policy" "dynamic_tiles" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.cloudfront.aws_cloudfront_cache_policy.dynamic_tiles,
│   on ..\..\terraform_modules\cloudfront\main.tf line 89, in resource "aws_cloudfront_cache_policy" "dynamic_tiles":
│   89: resource "aws_cloudfront_cache_policy" "dynamic_tiles" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.cloudfront.aws_cloudfront_function.cors_domain,
│   on ..\..\terraform_modules\cloudfront\main.tf line 112, in resource "aws_cloudfront_function" "cors_domain":
│  112: resource "aws_cloudfront_function" "cors_domain" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.efs.aws_efs_file_system.workhouse,
│   on ..\..\terraform_modules\efs\main.tf line 1, in resource "aws_efs_file_system" "workhouse":
│    1: resource "aws_efs_file_system" "workhouse" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.eventbridge.aws_cloudwatch_event_rule.stripe_events,
│   on ..\..\terraform_modules\eventbridge\main.tf line 4, in resource "aws_cloudwatch_event_rule" "stripe_events":
│    4: resource "aws_cloudwatch_event_rule" "stripe_events" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.eventbridge.data.aws_iam_policy_document.stripe_iam,
│   on ..\..\terraform_modules\eventbridge\main.tf line 28, in data "aws_iam_policy_document" "stripe_iam":
│   28: data "aws_iam_policy_document" "stripe_iam" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.lambda.data.aws_iam_policy_document.solr_indexation,
│   on ..\..\terraform_modules\lambda\main.tf line 3, in data "aws_iam_policy_document" "solr_indexation":
│    3: data "aws_iam_policy_document" "solr_indexation" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.lambda.aws_iam_policy.solr_indexation_policy_ec2,
│   on ..\..\terraform_modules\lambda\main.tf line 28, in resource "aws_iam_policy" "solr_indexation_policy_ec2":
│   28: resource "aws_iam_policy" "solr_indexation_policy_ec2" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.lambda.aws_iam_policy.solr_indexation_policy_sqs,
│   on ..\..\terraform_modules\lambda\main.tf line 60, in resource "aws_iam_policy" "solr_indexation_policy_sqs":
│   60: resource "aws_iam_policy" "solr_indexation_policy_sqs" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.lambda.data.aws_iam_policy_document.lambda_indexation_logging,
│   on ..\..\terraform_modules\lambda\main.tf line 161, in data "aws_iam_policy_document" "lambda_indexation_logging":
│  161: data "aws_iam_policy_document" "lambda_indexation_logging" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.lambda.data.aws_iam_policy_document.bulk_zoning_data,
│   on ..\..\terraform_modules\lambda\main.tf line 189, in data "aws_iam_policy_document" "bulk_zoning_data":
│  189: data "aws_iam_policy_document" "bulk_zoning_data" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.lambda.data.aws_iam_policy_document.bulk_zoning_data_policy,
│   on ..\..\terraform_modules\lambda\main.tf line 214, in data "aws_iam_policy_document" "bulk_zoning_data_policy":
│  214: data "aws_iam_policy_document" "bulk_zoning_data_policy" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.lambda.data.aws_iam_policy_document.bulk_zoning_data_s3_access_policy,
│   on ..\..\terraform_modules\lambda\main.tf line 362, in data "aws_iam_policy_document" "bulk_zoning_data_s3_access_policy":
│  362: data "aws_iam_policy_document" "bulk_zoning_data_s3_access_policy" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.lambda.aws_iam_policy.bulk_zoning_data_s3_access_policy_file_type["allowed_use_csv"],
│   on ..\..\terraform_modules\lambda\main.tf line 455, in resource "aws_iam_policy" "bulk_zoning_data_s3_access_policy_file_type":
│  455: resource "aws_iam_policy" "bulk_zoning_data_s3_access_policy_file_type" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.lambda.data.aws_iam_policy_document.gapi_reports,
│   on ..\..\terraform_modules\lambda\main.tf line 487, in data "aws_iam_policy_document" "gapi_reports":
│  487: data "aws_iam_policy_document" "gapi_reports" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.lambda.data.aws_iam_policy_document.gapi_reports_policy,
│   on ..\..\terraform_modules\lambda\main.tf line 511, in data "aws_iam_policy_document" "gapi_reports_policy":
│  511: data "aws_iam_policy_document" "gapi_reports_policy" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.lambda.data.aws_iam_policy_document.libreoffice_convertor_assume_role,
│   on ..\..\terraform_modules\lambda\main.tf line 711, in data "aws_iam_policy_document" "libreoffice_convertor_assume_role":
│  711: data "aws_iam_policy_document" "libreoffice_convertor_assume_role" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.lambda.data.aws_iam_policy_document.libreoffice_convertor_execution,
│   on ..\..\terraform_modules\lambda\main.tf line 729, in data "aws_iam_policy_document" "libreoffice_convertor_execution":
│  729: data "aws_iam_policy_document" "libreoffice_convertor_execution" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.lambda.data.aws_iam_policy_document.stripe_checkout_assume_role,
│   on ..\..\terraform_modules\lambda\main.tf line 863, in data "aws_iam_policy_document" "stripe_checkout_assume_role":
│  863: data "aws_iam_policy_document" "stripe_checkout_assume_role" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.lambda.data.aws_region.current,
│   on ..\..\terraform_modules\lambda\main.tf line 974, in data "aws_region" "current":
│  974: data "aws_region" "current" {}
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.lambda.data.aws_iam_policy_document.vector_tiles,
│   on ..\..\terraform_modules\lambda\main.tf line 986, in data "aws_iam_policy_document" "vector_tiles":
│  986: data "aws_iam_policy_document" "vector_tiles" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.lambda.data.aws_iam_policy_document.vector_tiles_policy,
│   on ..\..\terraform_modules\lambda\main.tf line 1010, in data "aws_iam_policy_document" "vector_tiles_policy":
│ 1010: data "aws_iam_policy_document" "vector_tiles_policy" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.lambda.data.aws_secretsmanager_secret.rds_secret,
│   on ..\..\terraform_modules\lambda\main.tf line 1055, in data "aws_secretsmanager_secret" "rds_secret":
│ 1055: data "aws_secretsmanager_secret" "rds_secret" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.lambda.data.aws_caller_identity.current,
│   on ..\..\terraform_modules\lambda\main.tf line 1109, in data "aws_caller_identity" "current":
│ 1109: data "aws_caller_identity" "current" {}
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.lambda.aws_lambda_layer_version.pdf_converter,
│   on ..\..\terraform_modules\lambda\main.tf line 1121, in resource "aws_lambda_layer_version" "pdf_converter":
│ 1121: resource "aws_lambda_layer_version" "pdf_converter" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.lambda.data.aws_iam_policy_document.pdf_converter,
│   on ..\..\terraform_modules\lambda\main.tf line 1166, in data "aws_iam_policy_document" "pdf_converter":
│ 1166: data "aws_iam_policy_document" "pdf_converter" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.network.aws_vpc.main,
│   on ..\..\terraform_modules\network\main.tf line 3, in resource "aws_vpc" "main":
│    3: resource "aws_vpc" "main" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.network.aws_internet_gateway.igw,
│   on ..\..\terraform_modules\network\main.tf line 18, in resource "aws_internet_gateway" "igw":
│   18: resource "aws_internet_gateway" "igw" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.network.aws_eip.ngw-b,
│   on ..\..\terraform_modules\network\main.tf line 31, in resource "aws_eip" "ngw-b":
│   31: resource "aws_eip" "ngw-b" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.network.aws_eip.ngw-e,
│   on ..\..\terraform_modules\network\main.tf line 41, in resource "aws_eip" "ngw-e":
│   41: resource "aws_eip" "ngw-e" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ReadResource call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.network.aws_vpc_dhcp_options.dhcp,
│   on ..\..\terraform_modules\network\main.tf line 167, in resource "aws_vpc_dhcp_options" "dhcp":
│  167: resource "aws_vpc_dhcp_options" "dhcp" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.shared_core1.aws_db_instance.pg_rds[0],
│   on ..\..\terraform_modules\rds\main.tf line 8, in resource "aws_db_instance" "pg_rds":
│    8: resource "aws_db_instance" "pg_rds" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.rds_sear1.aws_rds_cluster.a_mysql_rds[0],
│   on ..\..\terraform_modules\rds\main.tf line 91, in resource "aws_rds_cluster" "a_mysql_rds":
│   91: resource "aws_rds_cluster" "a_mysql_rds" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.shared_core1.aws_secretsmanager_secret.rds_secret,
│   on ..\..\terraform_modules\rds\main.tf line 151, in resource "aws_secretsmanager_secret" "rds_secret":
│  151: resource "aws_secretsmanager_secret" "rds_secret" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.rds_umt1.aws_secretsmanager_secret_version.rds_secret,
│   on ..\..\terraform_modules\rds\main.tf line 165, in resource "aws_secretsmanager_secret_version" "rds_secret":
│  165: resource "aws_secretsmanager_secret_version" "rds_secret" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.secrets_manager.aws_secretsmanager_secret.mailchimp_secrets,
│   on ..\..\terraform_modules\secrets-manager\main.tf line 1, in resource "aws_secretsmanager_secret" "mailchimp_secrets":
│    1: resource "aws_secretsmanager_secret" "mailchimp_secrets" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.secrets_manager.aws_secretsmanager_secret.stripe_secrets,
│   on ..\..\terraform_modules\secrets-manager\main.tf line 12, in resource "aws_secretsmanager_secret" "stripe_secrets":
│   12: resource "aws_secretsmanager_secret" "stripe_secrets" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.secrets_manager.data.aws_secretsmanager_secret.mailchimp,
│   on ..\..\terraform_modules\secrets-manager\main.tf line 23, in data "aws_secretsmanager_secret" "mailchimp":
│   23: data "aws_secretsmanager_secret" "mailchimp" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.secrets_manager.data.aws_secretsmanager_secret.stripe,
│   on ..\..\terraform_modules\secrets-manager\main.tf line 27, in data "aws_secretsmanager_secret" "stripe":
│   27: data "aws_secretsmanager_secret" "stripe" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.solr.data.aws_ami.solr_cloud_search_image,
│   on ..\..\terraform_modules\solr\main.tf line 2, in data "aws_ami" "solr_cloud_search_image":
│    2: data "aws_ami" "solr_cloud_search_image" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.sqs.aws_sqs_queue.solr_indexation_sqs_dead_letter_queue,
│   on ..\..\terraform_modules\sqs\main.tf line 35, in resource "aws_sqs_queue" "solr_indexation_sqs_dead_letter_queue":
│   35: resource "aws_sqs_queue" "solr_indexation_sqs_dead_letter_queue" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.state_shared.data.aws_ami.base_image,
│   on ..\..\terraform_modules\state\main.tf line 2, in data "aws_ami" "base_image":
│    2: data "aws_ami" "base_image" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.state_shared.data.aws_ami.base_image_22,
│   on ..\..\terraform_modules\state\main.tf line 11, in data "aws_ami" "base_image_22":
│   11: data "aws_ami" "base_image_22" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.state_shared.aws_security_group.solr1_sg,
│   on ..\..\terraform_modules\state\main.tf line 21, in resource "aws_security_group" "solr1_sg":
│   21: resource "aws_security_group" "solr1_sg" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.state_shared.aws_security_group.workhouse1_sg,
│   on ..\..\terraform_modules\state\main.tf line 99, in resource "aws_security_group" "workhouse1_sg":
│   99: resource "aws_security_group" "workhouse1_sg" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.state_shared.aws_route53_record.workhouse[0],
│   on ..\..\terraform_modules\state\main.tf line 141, in resource "aws_route53_record" "workhouse":
│  141: resource "aws_route53_record" "workhouse" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.web.data.aws_ami.web_image,
│   on ..\..\terraform_modules\web\main.tf line 5, in data "aws_ami" "web_image":
│    5: data "aws_ami" "web_image" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).ValidateDataResourceConfig call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.web.aws_route53_record.web,
│   on ..\..\terraform_modules\web\main.tf line 202, in resource "aws_route53_record" "web":
│  202: resource "aws_route53_record" "web" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵
╷
│ Error: Plugin did not respond
│
│   with module.web.aws_route53_record.subdomain[13],
│   on ..\..\terraform_modules\web\main.tf line 213, in resource "aws_route53_record" "subdomain":
│  213: resource "aws_route53_record" "subdomain" {
│
│ The plugin encountered an error, and failed to respond to the plugin.(*GRPCProvider).UpgradeResourceState call. The plugin logs may contain more details.
╵

Stack trace from the terraform-provider-aws_v6.18.0_x5.exe plugin:

panic: interface conversion: interface {} is nil, not string

goroutine 257 [running]:
github.com/hashicorp/terraform-provider-aws/internal/provider/sdkv2.(*sdkProvider).initialize.func4({0x23abcdf8, 0xc0032d5470}, 0x0?, {0x20cf39a0, 0xc0003b51d0})
        github.com/hashicorp/terraform-provider-aws/internal/provider/sdkv2/provider.go:748 +0x152
github.com/hashicorp/terraform-provider-aws/internal/provider/sdkv2.wrapResource.(*wrappedResource).stateUpgrade.func1({0x23abcdf8, 0xc0032d5470}, 0xc00350c0f0, {0x20cf39a0, 0xc0003b51d0})
        github.com/hashicorp/terraform-provider-aws/internal/provider/sdkv2/wrap.go:104 +0x99
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).upgradeJSONState(0xc001190660, {0x23abcdf8, 0xc0032d5470}, 0xd?, 0x20dc7b5f?, 0x14?)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.38.1/helper/schema/grpc_provider.go:642 +0x6c
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).UpgradeResourceState(0xc001190660, {0x23abcdf8?, 0xc0032d53b0?}, 0xc0032cd140)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.38.1/helper/schema/grpc_provider.go:515 +0x3f0
github.com/hashicorp/terraform-plugin-mux/tf5muxserver.(*muxServer).UpgradeResourceState(0xc001193200, {0x23abcdf8?, 0xc0032d50e0?}, 0xc0032cd140)
        github.com/hashicorp/terraform-plugin-mux@v0.21.0/tf5muxserver/mux_server_UpgradeResourceState.go:36 +0x190
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).UpgradeResourceState(0xc0006181e0, {0x23abcdf8?, 0xc0032d49c0?}, 0xc0032f6460)
        github.com/hashicorp/terraform-plugin-go@v0.29.0/tfprotov5/tf5server/server.go:799 +0x262
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_UpgradeResourceState_Handler({0x20bbc0a0, 0xc0006181e0}, {0x23abcdf8, 0xc0032d49c0}, 0xc0031fa900, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.29.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:699 +0x1a9
google.golang.org/grpc.(*Server).processUnaryRPC(0xc001be5400, {0x23abcdf8, 0xc0032d4930}, 0xc00329ea20, 0xc00275e840, 0x32994730, 0x0)
        google.golang.org/grpc@v1.75.1/server.go:1431 +0x1036
google.golang.org/grpc.(*Server).handleStream(0xc001be5400, {0x23abd8f8, 0xc002f09520}, 0xc00329ea20)
        google.golang.org/grpc@v1.75.1/server.go:1842 +0xb88
google.golang.org/grpc.(*Server).serveStreams.func2.1()
        google.golang.org/grpc@v1.75.1/server.go:1061 +0x7f
created by google.golang.org/grpc.(*Server).serveStreams.func2 in goroutine 24
        google.golang.org/grpc@v1.75.1/server.go:1072 +0x11d

Error: The terraform-provider-aws_v6.18.0_x5.exe plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```


### Sample Terraform Configuration

<details open>
<summary>Click to expand example of  configuration for one of the resources that fails</summary>

```hcl
# Create API Gateway Role
resource "aws_iam_role" "s3_api_gateway_role" {
  name = "s3-api-gateway-role"

  tags = {
    Role  = "api-gateway"
    Env   = var.env
    State = "shared"
  }

  # Create Trust Policy for API Gateway
  assume_role_policy = <<EOF
{
  "Version": "2012-10-17",
  "Statement": [
    {
      "Sid": "",
      "Effect": "Allow",
      "Principal": {
        "Service": "apigateway.amazonaws.com"
      },
      "Action": "sts:AssumeRole"
    }
  ]
} 
EOF
}
```
</details>


### Steps to Reproduce

1. Change version from hashicorp/aws version from 6.12.0 to 6.18.0
2. Run terraform plan

### Debug Logging

_No response_

### GenAI / LLM Assisted Development

_No response_

### Important Facts and References

_No response_

### Would you like to implement a fix?

No

--- a/.changelog/44860.txt
+++ b/.changelog/44860.txt
@@ -1,0 +1,3 @@
+```release-note:bug
+provider: Fix crash when setting override region during provider initialization
+```

--- a/internal/provider/sdkv2/provider.go
+++ b/internal/provider/sdkv2/provider.go
@@ -744,7 +744,7 @@ func (p *sdkProvider) initialize(ctx context.Context) (map[string]conns.ServiceP
 					var overrideRegion string
 
 					if isRegionOverrideEnabled && getAttribute != nil {
-						if region, ok := getAttribute(names.AttrRegion); ok {
+						if region, ok := getAttribute(names.AttrRegion); ok && region != nil {
 							overrideRegion = region.(string)
 						}
 					}

